client/src/components/DataFiles/DataFilesModals/DataFilesAddProjectModal.js

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback } from 'react';
```

Comment:
@jchuahtacc , what about renaming this to something like `DataFilesProjectAddModal`.  this can be ignored.  i'm just thinking it would be nice for the project-related modals to show up next when listed.  like:  `DataFilesProjectAddModal` `DataFilesProjectEditDescriptionModal` `DataFilesProjectEditMembersModal`

Reply:
Changed my mine on this so resolve if you want to.